install.sh

```diff
@@ -386,14 +379,16 @@ instlog "Installing ROS $(tr '[:lower:]' '[:upper:]' <<< ${ROS_VERSION:0:1})${RO
 sudo apt-get update -qq
 sudo apt-get install -qq ros-$ROS_VERSION-desktop-full
 
-# If ROS Indigo is being installed, break the metapackage and install an updated version of Gazebo
-if [[ "$ROS_VERSION" == "indigo" ]]; then
+# If ROS Kinetic is being installed, break the metapackage and install an updated version of Gazebo
+if [[ "$ROS_VERSION" == "kinetic" ]]; then
```

Comment:
Are you sure we need this now? I think the default kinetic gazebo is good as we used it on SubjuGator

Reply:
Removed as suggested.